tox.ini

```diff
@@ -3,4 +3,4 @@ envlist = py26, py27, py33, py34
 [testenv]
 deps = pytest
     mock
-commands = py.test
+commands = py.test --doctest-modules --doctest-ignore-import-errors
```

Comment:
If without `--doctest-ignore-import-errors`, raise 2 erorrs.

```
============================== ERRORS ===============================
__________________ ERROR collecting mycli/magic.py __________________
mycli/magic.py:2: in <module>
    import sql.parse
E   ImportError: No module named 'sql'
____________ ERROR collecting mycli/packages/counter.py _____________
mycli/packages/counter.py:4: in <module>
    from itertools import repeat, ifilter
E   ImportError: cannot import name 'ifilter'
===== 158 passed, 1 xfailed, 1 xpassed, 2 error in 0.37 seconds =====
```

I'm not familiar with `magic_function` of ipython, but `sql.parse` and `sql.connection` are right?